feconf.py

```diff
@@ -650,19 +650,26 @@ def get_empty_ratings():
 MAX_PLAYTHROUGHS_FOR_ISSUE = 5
 # Probability of recording a playthrough.
 RECORD_PLAYTHROUGH_PROBABILITY = 0.2
-# IDs of allowed types of issues.
-ALLOWED_ISSUE_IDS = [
+# Types of allowed issues.
+ALLOWED_ISSUE_TYPES = [
     'EarlyQuit',
```

Comment:
If you need to, you can also make these individual strings into named constants, and then have

ALLOWED_ISSUE_TYPES = [ISSUE_TYPE_EARLY_QUIT, ...]

Also do these constants really need to be in feconf? If they're being referenced by multiple modules or controllers then maybe that makes sense since that means they're in "shared" territory. But if their usage is fairly narrow then maybe they'd be better placed somewhere like stats_models instead.

Reply:
Done.